### PR TITLE
docs(ru): translation to challenges

### PR DESCRIPTION
**Pull request by stillst**

### CODE REVIEW

**Review by tomalaforge — APPROVED** [review]

**Comment by tomalaforge** [comment]
You need to update your project.

File: docs/src/content/docs/ru/challenges/angular/8-pipe-pure.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+description: Испытание 8 про создание чистого пайпа
+author: thomas-laforge
+challengeNumber: 8
+command: angular-pipe-easy
+blogLink: https://medium.com/ngconf/deep-dive-into-angular-pipes-c040588cd15d
+sidebar:
+  order: 3
+---
+
+Цель этой серии из 3 задач на тему пайпов - освоить работу с **пайпами** в Angular.
```
Comment (svenson95) [inline docs/src/content/docs/ru/challenges/angular/8-pipe-pure.md:12]: The docs for this challenge was updated after your pull request, if possible please check out this challenge again and update the translation.

Comment (stillst) [reply docs/src/content/docs/ru/challenges/angular/8-pipe-pure.md]: Sure. Did it here https://github.com/tomalaforge/angular-challenges/pull/609

**Comment by stillst** [comment]
> You need to update your project.

I accidentally closed this pull request 🙈. There is another https://github.com/tomalaforge/angular-challenges/pull/609

**Pull request closed without merging**